packages/test/tsconfig.json

```diff
@@ -13,6 +13,6 @@
     { "path": "../worker" },
     { "path": "../workflow" }
   ],
-  "include": ["./src/**/*.ts", "./src/history_files/*.json"],
+  "include": ["./src/**/*.ts", "./history_files/*.json"],
```

Comment:
Oh now I see why you put this in `src`, just so you can import JSONs directly...
We don't have any json histories AFAICT so we could probably just remove this line.
I don't like using sources outside of the `src` directory because it means that anything that was in `src` will be outputted to `lib/src`